rubygem-dynflow/rubygem-dynflow.spec

```diff
@@ -5,12 +5,12 @@
 
 Summary: DYNamic workFLOW engine
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.8.13
+Version: 0.8.15
 Release: 1%{?foremandist}%{?dist}
 Group: Development/Languages
 License: MIT
 URL: https://github.com/Dynflow/dynflow
-Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
```

Comment:
Use an HTTP source here for now, git-annex on our builders don't currently support HTTPS (should be fixed soon once rebuilt). The previous comment was only about the URL field, not Source0.